### PR TITLE
[FEAT] Add human sign on Goblin Town

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

This PR will add the human sign on Goblin Town.

Screen Shot 2022-05-08 at 1 23 28 pm

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Pranav-yadav** [comment]
Don't merge this before merging `#827` 😆 .

Btw, It looks better now.. 🌈 
PS: Don't forget to review `#827` :)

File: src/features/goblins/components/TownEntryModal.tsx
Comment (adamhannigan) [inline src/features/goblins/components/TownEntryModal.tsx:19]: This may cause issues in the future if we add additional children routes to Goblin town. Non-blocking, but consider in the future being explicit of returning to the /farm route

**Review by adamhannigan — APPROVED** [review]
Nice work mate 💪

**Pull request merged**
